src/Components/PastMenu.js

```diff
@@ -0,0 +1,254 @@
+//React components
+import React from "react";
+import Carousel from "react-bootstrap/Carousel";
+
+//indochine images
+import chickpeaTofuSalad from "../images/indochine/starters/chickpea-tofu-salad.jpg";
```

Comment:
I think the real challenge for this website is on defining an architecture and data structures, that allow dynamic imports for relevant dishes here! Doing all these imports manually seems very tedious and also does not allow for easy extension!